pyvcloud/system_test_framework/environment.py

```diff
@@ -592,6 +594,42 @@ def create_direct_ovdc_network(cls):
         cls._sys_admin_client.get_task_monitor()\
             .wait_for_success(task=result.Tasks.Task[0])
 
+    @classmethod
+    def create_routed_orgvdc_network(cls):
+        """Creates a routed org vdc network.
+
+        The name of the created org vdc network is specified in the
+        configuration file, skips creating one, if such a network already
```

Comment:
This would be 'ovdc_network_constant.py' constants file. not the configuration file.